### DISCUSSION TITLE
Can you remove frequencies from auto populating on sats?

### DISCUSSION THREAD

**Discussion by ant-lefebvre** [magicbug/Cloudlog#1080] [Q&A] [answered]
I have used TQSL in the past to upload sat contacts to LoTW and never enter a frequency. But it seems that cloudlog auto populates a frequency with every QSO. Is there any way to turn that off and leave it blank?

**Comment by magicbug** [comment, marked as answer]
Hi Ant,

Currently no, we populate the center freq, that said shouldn't cause any matching problems.. if there's a real desire to turn it off I'll add it to the todo list.

Peter.
